src/posix/main.c

```diff
@@ -141,6 +141,7 @@ enum
     OT_POSIX_OPT_DRY_RUN                 = 'n',
     OT_POSIX_OPT_HELP                    = 'h',
     OT_POSIX_OPT_INTERFACE_NAME          = 'I',
+    OT_POSIX_OPT_PERSISTENT_INTERFACE    = 'p',
```

Comment:
If persistent is better in most cases, I don't think we need add an option for not enabling it.

Reply:
Maybe it's still a good idea to add an option. 
Some users may still prefer temporary interface?

Reply:
Agree, This is not fully compatible with previous behavior.